Packages/nadena.dev.modular-avatar/Runtime/ModularAvatarObjectRemover.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+
+namespace nadena.dev.modular_avatar.core
+{
+    [ExecuteInEditMode]
+    [AddComponentMenu("Modular Avatar/MA Object Remover")]
+    public class ModularAvatarObjectRemover : AvatarTagComponent
+    {
+        public bool hideInHierarchy = true;
+        public bool keepDisabled = true;
+        public bool removePrefabComponents;
+        public List<GameObject> objectsToRemove = new List<GameObject>();
+        private List<GameObject> _lastObjectsToRemove;
```

Comment:
Should this be serialized, so that changes that happen outside of the inspector are reverted properly?

Reply:
ONValidate then should still handle it